fastlane/lib/fastlane/actions/latest_testflight_build_number.rb

```diff
@@ -69,6 +33,12 @@ def self.available_options
         user ||= CredentialsManager::AppfileConfig.try_fetch_value(:apple_id)
 
         [
+          FastlaneCore::ConfigItem.new(key: :live,
+                                       short_option: "-l",
+                                       env_name: "CURRENT_BUILD_NUMBER_LIVE",
+                                       description: "Query the live version (ready-for-sale)",
+                                       optional: true,
+                                       default_value: true),
```

Comment:
Let's do it like this: Make the default value `false`, and not set it on line 11 (just remove line 11)